### ISSUE TITLE
What about a new name

### ISSUE THREAD

**Issue by adaur** [SickRage/sickrage-issues#26]
I think we all agree that we need a new name.

Let's think about what it could be, whether you're a member of the team or not :)

**Comment by adaur** [comment]
I'll start with **AutoTV**

**Comment by maxcore** [comment]
GrabTV

**Comment by medariox** [comment]
I like **Dionysus**. He was the Greek god of Theatre. I think it's an interesting name and not too hard to remember.

**Comment by adaur** [comment]
I thought about GrabTV too, but finding a good domain name will be hard.

Dionysus, why not, but I'm not sure how we could make it up to the 1st page of Google...

**Comment by labrys** [comment]
I like ODIN, and it doesn't hurt that its similar to KODI but not particularly tied to it.

**Comment by labrys** [comment]
And too many possible spellings of Dionysus  (original is closer to Dionysius)

**Comment by adaur** [comment]
**ShowGrabber** ? sounds like Showrunner :)

**Comment by medariox** [comment]
@labrys 
Dionysus seems to be the most commonly used spelling. http://www.britannica.com/topic/Dionysus

**Comment by victoriasavi** [comment]
If you are going with a new name it might be an idea to do so with a new stylesheet as well, to differentiate from the old project.

**Comment by adaur** [comment]
I agree, but it will take some time

**Comment by hernandito** [comment]
Great idea on changing the name. I like the legacy sound... it is all based on the original Sickbeard, and anything "Sick" is a tribute and a continuation of things.

Ideas:
SickBalls
SickBells
SickGear (I think its taken right?)
SickTV
BeardRage
RageTV
RageOn
HipRage

And PLEASE, change the logo... I cannot take the middle-school notebook shark cartoon sketch any longer... Something like the below.

http://i.imgur.com/WURm0Gm.png

Also, to differentiate further, please take a look at adding the view I mention on this post: I also did some css tweaks which are in a download in my link below.

https://sickrage.tv/forums/forum/main-category/feature-requests/28888-req-a-new-medium-poster-view-for-home-page-see-my-screencap

**Comment by OmgImAlexis** [comment]
I agree with @victoriasavi, maybe we can wait till bootstrap4 is added and once that's done we'll release it and the new name?

**Comment by OmgImAlexis** [comment]
@hernandito I'm really liking that, if you want to add a PR I can have a look at merging it. :+1:

**Comment by maxcore** [comment]
@hernandito nice design!!

**Comment by miigotu** [comment]
Few things about this:

The name will stay as it is for some time, to help users make the connection that this is not a fork but a migration. 

The name is not being switched because of the switch to a new github Org. A name change was already planned in the near future anyways even if the migration didnt happen. The reasoning for a new name is mostly because `SickRage` doesnt make sense anymore since `TVRage` went down.

That said, we SHOULD start thinking about a new name.

Here are some criteria suggestions should meet:

Anything including sick, beard, rage, TV, pirate, or any reference to a 3rd party site/service/application is auto-rejected. This is because we want a distinct memorable identity, which does not rely on the success of a 3rd party to continue making sense for the forseeable future. Also, TV is ruled out because we do not plan to limit the future development and features to just TV. The possibility of movies or other medium must remain open so the name still makes sense should we add say movie support. Anything referencing pirates is rejected to avoid unnecessary attention from anti-piracy groups.

These criteria are subject to change, but is fairly sufficient I think.

Now that we have that out of the way, suggest to your heart's content. Suggestions which are easily iconified are great, because if we decide on a name and image before the actual name change the icon can be changed early. The shark blows.

**Comment by phaseburn** [comment]
Dionysus sounds great to me, but I do think that it'd be hard to get into any relevant search results in and of itself. As such, maybe something like "Dionysus Screens" or some other second word relating to what the app does (with out mentioning sick, beard, rage, TV, pirate, usenet, torrent, etc). Maybe Dionysus DM (Download Manager), something that is likely to be unique when paired with the name...

**Comment by OmgImAlexis** [comment]
Honestly I don't think we should be going for anything that isn't easy to say. Dionysus isn't only had to pronounce especially for non-english speakers but also hard to spell.

**Comment by miigotu** [comment]
@phaseburn I have updated my comment with criteria updates from your comment.
@adaur hijacking your O.P. now with my comment.

**Comment by miigotu** [comment]
Rejecting ODIN, it is a software for flashing android fastboot images on linux.

**Comment by miigotu** [comment]
MediaSwagger
~~Media360~~ (Taken)

**Comment by adaur** [comment]
@miigotu no problem :)

Dionysus wouldn't make it easy for users to find us.

MediaSwagger : not a fan since "swag" is has a very bad connotation, at least where I live. But I like the 'media' prefix.

**Comment by miigotu** [comment]
MediaSnoop
VideoSnoop
MediaGadget

(I have lots of stupid ideas, just throwing all out there in case they stick)

**Comment by labrys** [comment]
How about ExMachina  or XMachina as in Deus Ex Machina, the theater plot device.   Lit. From the machine.

**Comment by labrys** [comment]
@miigotu Snoop has a bad connation too, and MediaGadget seems overly generic.

**Comment by OmgImAlexis** [comment]
MediaBox ?

**Comment by labrys** [comment]
MediaBox ain't too bad... still fairly generic though.

**Comment by OmgImAlexis** [comment]
We really need something simple and easy to type. Things like `ExMachina` sound nice but how often is it actually going to be spelt correctly? Especially for new users trying to find us.. :disappointed:

**Comment by labrys** [comment]
That's why I was thinking XMachina might be better... even mispelled like XMachine it should still return results via Google

**Comment by labrys** [comment]
Or maybe not Xmachine already exists :(

**Comment by OmgImAlexis** [comment]
@labrys yup just checked google. :disappointed:

**Comment by labrys** [comment]
but MediaBox while it would likely be spelled correctly would return tooo many things as well.. too generic

**Comment by OmgImAlexis** [comment]
@labrys jump on IRC

**Comment by labrys** [comment]
im on irc

**Comment by labrys** [comment]
OmgImAlexis what channel

**Comment by labrys** [comment]
nm irc client was locked up

**Comment by OmgImAlexis** [comment]
What about rebranding the org as ZenMedia and the program as Zen.

**Comment by labrys** [comment]
Or Broadcast / BroadCast to match up with CouchPotato HeadPhones etc..

**Comment by adaur** [comment]
MediaGod
MediaChef (and we can change the shark to a chef and have puns about cooking) - I like this one
MediAssistant (probably too close to medication and medecine area)
Mediasy like Media + Easy

**Comment by fernandog** [comment]
I would keep our origin: SickMaze - supporting tvmaze.

**Comment by miigotu** [comment]
@fernandog we really dont want to use a service as the basis for our name. If tvmaze happens to go down (and tvmaze isnt even going to be the only extra indexer we implement) then we are once again stuck with having to change the name.

**Comment by fernandog** [comment]
i know. I just like the name. lol

SickMaze, amazing software! lol

**Comment by duramato** [comment]
I would say something with flix in the name to represent the shows and movies(if whe support) 
Ie SickFlix

**Comment by adaur** [comment]
Do you want to keep the sick thing? Not very positive

**Comment by labrys** [comment]
I don't think we really want to keep the sick part...

**Comment by duramato** [comment]
I'm not saying to keep the sick ,it was just an example

**Comment by adaur** [comment]
Mediaflix
Flixmedia
Flowflix
Flixflow
Downflix
Flixdown
Flixplay
Playflix
Helloflix
Hiflix
Highflix

**Comment by jstevenson72** [comment]
SlickFlix, or SlickShows, or SlickMedia.  It's easy to say, and expresses how easy the software makes getting content.  The software literally changed my family's life.

**Comment by miigotu** [comment]
FlixMaster

**Comment by fernandog** [comment]
liked SlickShows lol

**Comment by miigotu** [comment]
Shows implies we only will do shows ever

**Comment by jstevenson72** [comment]
When going to a movie, we say "What time is the Show?".  Or we search for Movie Show Times on Google.  Show just meaning performance, whether TV, Movie, Play, etc.

**Comment by miigotu** [comment]
In the US we dont say that, maybe "what time is it showing?", but a show is not a movie or vice versa

**Comment by adaur** [comment]
Ripflix

**Comment by strike84** [comment]
I personally think the name should be a one-word name like KODI, because it's simple and tends to stick in mind almost right away. And it shouldn't be limited to something that the software does or represents. Most of the top memorable unique names have nothing to do with the product itself. And it doesn't have to be all that fancy, APPLE is a fruit for god's sake but it's still one of the top companies in the world. Just some tips for the folks trying to come up with names. I couldn't think of so many one-word names myself and I have no creative imagination what so ever but here are my suggestions. I'll probably throw in some more at a later time. Btw I haven't googled any of these names so I don't know if they're taken.

ORBIT
SnatchWatch
MediaCORE
MediaCast 

ORBIT Media
ORBIT Vision 
Media Reactor
Media Horizon
Media Scope

**Comment by alisonatwork** [comment]
I also agree that it helps an awful lot of a google search if you have a one-word, unique name like Kodi.

On the other hand, I like the idea of a name that is slightly descriptive but also metaphorical. **Media Cowboy** sticks in my head - it's your personal white-hat who works all day corraling the wild media. Or **Video Vaquero**, for the alliteration fans. **Media Shepherd** has perhaps a little less wild west romance, but may be something else to bounce off.

**Comment by maxcore** [comment]
**TVDinner**

;)

**Comment by labrys** [comment]
lol @maxcore

**Comment by VinceVal** [comment]
MediaVac
MediaVacuum

**Comment by VinceVal** [comment]
MediaMagnet
MediaFlux

**Comment by BenjV** [comment]
How about 'Capto' or 'AutoCapto'

Capto is Latin for Grab

**Comment by ikriz** [comment]
**Out of the box:**
Storytime
Projector

**In the box:**
MediaGuide
MediaTube
BoxTube
BoxOffice
MediaBug

**Nostalgia:**
MediaAmp
Napsteria

**Others:**
Frosting (this is funny)
Transient (even better)

**Comment by miigotu** [comment]
BitCrawler
BitSmack
BitSnack
SceneCrawl
PotatoHead
Tater
PotatoBud

PopcornCMS
HomeCMS
TheaterCMS

Yodo

**Comment by hernandito** [comment]
SickWrangler
SickAngel (Would love to try icons on that one - Sickbeard guy with a halo)
TheMadness
SickMedia
SickServer
SickPirate
SickSick
SickPick
SickPack
SickSh*t

**Comment by hernandito** [comment]
@ OmgImAlexis . I dont have anything in a github... I manually edited the files in my server. The edited files are in my download. But if you rather I fork the repo, and add the edited files, and do a PR, I can do tha. Just let me know.

**Comment by adaur** [comment]
No sick anymore please

**Comment by VinceVal** [comment]
> But if you rather I fork the repo, and add the edited files, and do a PR, I can do tha. Just let me know.

@hernandito, yes, please.

**Comment by Arcanemagus** [comment]
@hernandito:

"***Anything including sick, beard, rage, TV, shows, pirate, torrent, usenet, nzb or any reference to a 3rd party site/service/application is auto-rejected.***"

**Comment by hernandito** [comment]
Got it...
illFacialHair?  Just kidding.
 : )

**Comment by hernandito** [comment]
@OmgImAlexis @VinceVal I have submitted the pull Request. Please test it around if you can. I am NOT an expert coder. I can tweak CSS and some minor things. I am sure my layouts will need polishing.

**Comment by phaseburn** [comment]
Telegenie

A play on "telegenic", which describes something's appeal on television to an audience.

**Comment by miigotu** [comment]
TeleGenesis

**Comment by adaur** [comment]
So you don't want TV in the name but Tele is allowed? ;)

**Comment by phaseburn** [comment]
@adaur "Tele" is fairly modern common root wood in English, telegraph, telephone, television, etc, and simply means "transmission" or "over a distance". There's somewhere north of 200+ words that use it. Totally think tele-names are fine, where as TV is less so because of the reasons @miigotu stated.

**Comment by fernandog** [comment]
mediaSnatcher

**Comment by miigotu** [comment]
BitSnatch, FlixSnatch, FlicSnatch

**Comment by djack42** [comment]
Add [Once], Sit Back and Enjoy (AOSBE or ASBE)

I really like that one, it's like kodi, short and easy to remember, meaningfull and there isn't TV related result on google.
After all, that's just what we'd like to do, add the show, sit back and enjoy... (There's room for improvement.)

**Comment by phaseburn** [comment]
@miigotu Where I come from, "snatch" is a rather unfriendly term related to a lady's unmentionables, and, BitSnatch makes me think of all the pixelated out lady parts from certain Japanese Hentai videos :-P Lets please not go that where unless you want a cartoony 8-bit <censored> as your icon :-D

**Comment by miigotu** [comment]
MetaBit, BtaBite

**Comment by phaseburn** [comment]
VidBytes

**Comment by theskipster** [comment]
Media-Vac
MoreMedia
PressRecord
PVRPicker
MyMediaThing

**Comment by labrys** [comment]
Possum Media Server:  Because the possum-bilities are endless!

**Comment by MGaetan89** [comment]
So its acronym would be PMS? :tongue:

**Comment by labrys** [comment]
:)

**Comment by labrys** [comment]
###### And how would people describe it?     

It's Awesome Possum! 
![AWESOME]
(https://pbs.twimg.com/profile_images/559362705870430209/3Pkj_pjq_400x400.jpeg)

**Comment by labrys** [comment]
But on a more serious note i was thinking  **MediaRes** as in [In Media Res](https://en.wikipedia.org/wiki/In_medias_res) (lit. into the middle of things) but also as in Media Resource, Media Resolver, and Media Resolution

**Comment by kendog819** [comment]
+1 to SlickFlix

**Comment by miigotu** [comment]
You guys give up? lol

**Comment by djack42** [comment]
Just pick one from a random generator already !
After all, whatever the name, some will like it and some will dislike it.
And each one will have their own reasons...
Trying to choose the one liked by everyone wont do IMO.

**Comment by adaur** [comment]
I think we already have more than enough propositions :)

**Comment by miigotu** [comment]
None that SCREAM yes though lol. Of course we can find one that is OK here, but I'm hoping we stumble onto something magical.

**Comment by hernandito** [comment]
Inspired by above comment, I like:
"SomethingMagical"
SumMagic
TeleMagic
SomeWonder
10thWonder
Obelisc (i know its misspelled - tribute to 2001 Space Odyssey - lots of + connotations)
Must of above domains available as .org and/or .net.

**Comment by strike84** [comment]
So seems like I got "media" stuck in my head....

MediaScout
MediaWizard
MediaSpot
MediaSupply
SmartMedia
MediaHub
MediaZone
MediaLink
MediaFrenzy

IM (Instant Media or Intelligent Media)

ShowRoom (kinda like this one, can still use SR too.. =))

**Comment by strike84** [comment]
Spongy "Just absorbs your media"

**Comment by hernandito** [comment]
IMHO Anything Media, Bit, Meta, Snatch, Video, Flick, Hub, Wizard, Zone, etc are over used cliche terms. I like how SickBeard and SickRage are quite different and uncommon word mashups.

How about some more inspiring, or intrigue combinations;

RedTower
FirstPier
FifthColumn
FastHelix
Heisenberg
ThirdRail
7thSense
DarkDilithium
Trilithium
Plutonium7 (that one is going to get me on some gov. list)

I can go forever...

**Comment by miigotu** [comment]
Gopher

**Comment by OmgImAlexis** [comment]
RedTower sounds nice and we could have a logo in the same style as Travis-CI with that nice cartoony look.

**Comment by miigotu** [comment]
I like Gopher a lot lol... "Gopher this, Gopher that" Go for this....

**Comment by theskipster** [comment]
BlueBox

Think, TARDIS. It kinda goes back in time to find old shows, and, knows lots about what is coming, but can't tell us.

**Comment by hernandito** [comment]
I like the idea of  Presence. Always there... The following domains are available:
RedPresence.com 
MyPresence.org

I am sure others can come up with some good combos on that.. a Ghost icon type logo...

Gopher.life and gopher.io are available... I like that as well.

**Comment by labrys** [comment]
@miigotu While I prefer Possum, I'd be cool with gopher :)

**Comment by Thraxis** [comment]
Gopher....  "We'll dig those shows up for you"

**Comment by medariox** [comment]
How about **Meerkat** or **Suricate**?
Cause, you know, they seem to be always looking for something...
![Meerkat](https://s-media-cache-ak0.pinimg.com/236x/87/61/66/8761660ffbfe8bdd9d61776e8142d6b4.jpg)

**Comment by xios01** [comment]
I saw earlier the suggestion for ShowRunner... i really liked that suggestion... 

However, playing with the idea behind channel/TV surfing...

ShowSurf
TVSurf
SickSurf (eh, had to throw it out there ;-) )
SurfIt

etc...

**Comment by ikriz** [comment]
Please anything but an animal... bye bye unique search results, not to mention all the other projects that use animal names.

Something either totally not related to media (not an animal) or something that touches the subject.

"hmm, i'll check gopher if my show is in..." no thanks

i'd rather check Candyfloss or Frosting if my show is in

### Candyfloss
or just
### Candy

it's upthere with popcorn ;)

**Comment by labrys** [comment]
@ikriz Says the person with a Kiwi as their profile pic :grinning:

**Comment by duramato** [comment]
@ikriz CandyFloss makes my mind think of dentalfloss lol

**Comment by adaur** [comment]
Candy: how do you expect Google to find us?

**Comment by xios01** [comment]
isn't Gopher taken?

https://en.m.wikipedia.org/wiki/Gopher_(protocol)

**Comment by miigotu** [comment]
It would likely be "Gofer", and even the protocol "Gopher" is based on the same slang.
https://en.m.wikipedia.org/wiki/Gofer

"A gofer, go-fer or gopher /ˈɡoʊfər/ is an employee who specializes in delivery of special items to their superior(s). The special items may be anything from a cup of coffee to a tailored suit or a car. Outside business the term is used to describe a child or young adult who is learning how to do tasks and is sent to fetch items."

**Comment by Thraxis** [comment]
Fido?  But regular e-mail is faster...

**Comment by duramato** [comment]
Ambrosio

**Comment by Yellow1144** [comment]
What about:

Pluto or Pluton

God of the underworld, the underworld of TV downloading. Also the god of Wealth

Maybe..

Jupiter...

Getting ideas from Roman gods. There are a lot of others to choosefrom

**Comment by alisonatwork** [comment]
I must admit Gofer is kinda growing on me, though it might be worth adding a unique prefix or suffix to make it easier to google. I agree with @djack42 though, no matter what the name ends up being some people will love it and some people will hate it. The point is they'll still use the software if it's better than the alternatives. I still use Kodi even though i thought XBMC as a name had historic charm. I didn't like the Sick* names, but here i am, because it's a tool that does its job well. ¯\_(ツ)_/¯ As long as there is a good migration strategy for end-users and OS package/port maintainers, i'm happy to spin the bottle on a few top options.

**Comment by pyrodex** [comment]
SnatchMaster. It is ambiguous enough to work with TV, Movies, etc...

**Comment by labrys** [comment]
@pyrodex I sure TV and Movies are the first thing that would come to mind for that.

**Comment by hernandito** [comment]
ZipperHead - my icon interpretation.
![zipperhead](https://cloud.githubusercontent.com/assets/787317/11610402/4c8c4116-9b70-11e5-9848-dd4c597af7ed.png)

**Comment by miigotu** [comment]
@pyrodex tell her to close her legs lol

**Comment by Theli93** [comment]
While I was somewhat intrigued with the idea of Dionysus (the original god of wine, women and song), he does have a close cousin (so to speak) in Pan (the god). This also has the advantage of meaning "all". To add a little whimsy, it could be PanHandle... he handles it all!

Staying on theme, I think I'm a little more inclined to another figure. By way of retaining the ideas of "media" and "usenet" (or "user") we could have Medusa! Her head makes for a great icon (as it has for centuries) and all those blasted snakes are good for grabbin' on to all sorts of things!

**Comment by miigotu** [comment]
I like Medusa lol, but the google search though....

**Comment by labrys** [comment]
Yeah I was thinking medusa myself but thought google might be a problem.  Maybe combined with another term such as MedusaMedia or something.  Also Antikythera as in the [Antikythera Mechanism](https://en.wikipedia.org/wiki/Antikythera_mechanism) which is an ancient analog computer.  The icon could be a clockwork gear type assembly.

**Comment by Theli93** [comment]
Okay, well I considered Medusa Manager, but it appears there may be some obscure use there. Taking a step back from the name you could go with Gorgon or Gorgon Manager. Gorgons were Medusa  and her two sisters. If the project expands significantly, this original set of functions could be INTERNALY "Medusa", providing at least two more code "monsters" to expand into. ("Gorgon Manager – Medusa")

**Comment by Theli93** [comment]
Playing off acronyms; PVC (Personal Video Capture) in opposition to a PVR. PVC also being a type of plastic. It could also be rendered as PyVC.

**Comment by labrys** [comment]
@Theli93 and in that vein could also go with Moirai - the three fates.  With them spinning, measuring, and cutting the thread of life.  Although personally I prefer medusa.

**Comment by miigotu** [comment]
meducifer (devil medusa?) lol
Medichi/Medici

L33tch

3p1cL33t3ch310nk1773r?!?!11

**Comment by labrys** [comment]
Sticking with the Greek mythology theme there's Ariadne who helped Theseus defeat the Minotaur by providing him with the the tools he needed.  Thread that allowed him to overcome the labyrinth and a sword to vanquish the Minotaur, and thus freeing Athens from slavery.

**Comment by medariox** [comment]
Great name **Ariadne**, you have my support @labrys!

**Comment by labrys** [comment]
And hey we can always go with Outis (aka Nobody).  As in what Odysseus called himself when he blinded the Cyclops.  So when everyone asked the Cyclops who blinded him, all he could do was wail "Nobody did this to me"

**Comment by Theli93** [comment]
Playing off the "snatching"; there's the infamous Bandersnatch, of Lewis Carroll fame - ref: [Wikipedia: Bandersnatch](https://en.wikipedia.org/wiki/Bandersnatch) - and it is known to have" a long neck and snapping jaws", and is described "as ferocious and extraordinarily fast"
If the "Bander-" is thought to be misleading it could be modified to "Tele-" snatch (having the connotations of both television and distance (i.e. tele-), but using a modified bandersnatch image a the logo

**Comment by Thraxis** [comment]
If we're going Lewis Carroll why not just go with...

Vorpal

**Comment by shovelhed** [comment]
Another good god name would be Poseidon. Great opportunity for a new logo. God of the sea.

**Comment by Theli93** [comment]
Hey, how about Janus, the god with two faces? The google looks pretty clean. Good icon; clean and simple. The two faces would have double significance: 1) looking forward and back (backlogs and future events) and 2) the single head looking for both nzbs and torrents! Further descriptor could be added for clarification (ie. Janus BinBit, or some such). Using the head only as icon (as is typical) would also reflect that is sees, and communicates, but takes no action of its own (ie. it doesn't download, just seeks the information and communicates it to a laborer).

**Comment by labrys** [comment]
@Theli93 I like the significance of Janus, not so sure I care for the name personally.  Also I think we should exclude it since its already used in a media context.  It's the name of a [DRM platform](https://en.wikipedia.org/wiki/Janus_%28DRM%29).

**Comment by Theli93** [comment]
@labrys Dangit... didn't see that! <inaudible miscellaneous grumbling curses>

**Comment by davidsodre** [comment]
Easy Media Grabber - EMG

**Comment by miigotu** [comment]
Grab my media, lol, and while you're down there.....

**Comment by Velocus** [comment]
What about
TVSAURUS
TVREX

**Comment by Velocus** [comment]
XFiles. LOL

Oh damn. Taken.

**Comment by reaxi0n** [comment]
Not to derail the discussion but this comment: "And PLEASE, change the logo... I cannot take the middle-school notebook shark cartoon sketch any longer..."

AMEN to that. That 1980 MS clipart needs to go! It looks terrible.

**Comment by miigotu** [comment]
I agree lol, the "muscle sharks rawr" is akin to marketing cigarettes to 12 year olds.

**Comment by adaur** [comment]
Don't touch my shark! :p

**Comment by Trotter73** [comment]
How about Videodrome ?

It was an early 80's David Cronenberg film about a TV Executive, quite apt I think, looking at google the only references are to the film so anything else would stand out....

**Comment by miigotu** [comment]
MediaDrone

**Comment by Amleto** [comment]
drone\* or \*drone just makes me thing of sonarr...

**Comment by Yellow1144** [comment]
Sick 'Stache - Give the original SB logo a handlebar moustace instead of the Beard lol

**Comment by miigotu** [comment]
https://play.google.com/store/apps/details?id=org.sickstache&hl=en

**Comment by Yellow1144** [comment]
LOLOLOL!!!!!!! No Kidding.. dammit..idea just a little too late

**Comment by identd113** [comment]
NetPVR or PyVR

**Comment by miigotu** [comment]
SR is NOT a PVR.

**Comment by frriction** [comment]
Something related to Physics.
DOPPLER
FREQUENCY
GAMMA
~~Optic~~
PHOTON
quark
quantum 
~~wave~~
fluorescence

**Comment by frriction** [comment]
resonance

**Comment by identd113** [comment]
Entropy
Arda
Encompass

**Comment by bartley69** [comment]
I dig SlickFlix

**Comment by labrys** [comment]
Is it just me or does SlickFlix make you think of a slimy booger you're trying to get off your finger?

**Comment by trendless** [comment]
Garb or Barge or something like; anagrams of 'grab'. Could mess with the spelling, eg 'Barj'. The latter has especially good visual and contextual imagery that could associates well with the software's functionality

Or something with Octopus (or some concatenation thereof, eg 'Groktopus'. Fun animated characters possible with this, from a logo standpoint

Or something that integrates SickRage's initials (SR). ScouRge, Skurj, ScouR, Scower, Supr, Skraip(e). Some nuanced/understated visual continuity would be possible with the letters S and R, and to a lesser extent, the K.

Or you could go with some sort of national stereotype, eg Syrup and the accompanying Canadian shtick, Pizza/Italian, Rasta/Jamaican, СМИ/Russian, Great Piratewall/Chinese. Heck, even branding it as some sort of satire of North Korea (KimJongTorr is terrible, but gives you an example of where I'm going) would give you a ton of leeway to create a great design identity

**Comment by labrys** [comment]
Along the same lines as Medusa, go with a Hydra reference such as Lernaean Library Manager.

**Comment by Zzaap** [comment]
SeekFlix: it keeps a reference to old name phonetically only, and it makes sense.

**Comment by kendog819** [comment]
Can we setup a poll and vote? Like this: http://www.poll-maker.com/poll545992xb3594401-23

I'm ready for a name change, whatever it may be. I can add all the options to it that have been mentioned so far.

**Comment by miigotu** [comment]
Not time yet

**Comment by mu-stash** [comment]
The name you are looking for is **Mustash**

Because mustache is a beard evolution
Because Mustash is my media stash
New logo should come easily.

**Comment by miigotu** [comment]
@mu-stash, nah, too hip =P

**Comment by mrNobody1337** [comment]
I would suggest LiveOver, since that is what happens when you use this program xD Icon can be a dead guy in front of a tv :p (j/k guys 'n girls)

I don't have any good name suggestions, but I would like to say that I think anything with flix is a bad idea because of netflix.

**Comment by duramato** [comment]
@mrNobody1337 http://www.urbandictionary.com/define.php?term=flix

**Comment by mrNobody1337** [comment]
Yeah I know the definition, but it's about what people associate flix with. I think netflix has a strong brand and newbies might get confused. Or perhaps I'm just underestimating new users xD

**Comment by Antioch18** [comment]
I liked the proposal with "bit" mentioned earlier.

BitGrab
BitGrabber
BitNab
BitNabber
BitRage
BitBurn
BitStorm
BitStream
NullBit

**Comment by dmarcelino** [comment]
Taking the shark inspiration I suggest **kokala** after Kane-i-kokala:

> Kane-i-kokala: This was a shark god that was revered for taking shipwreck victims to safety. His sacred fish was the kokala and those that believed in his existence would not touch or eat this fish for fear of displeasing him.

Not too many google results on it and kokala.tv is free. It's also not too hard to pronounce or spell and it's fairly distinct.

Anyway I think soon we should do one of two things:
a) Vote on all name suggestions and from the top 10 the SickRage team selects the final name; or
b) The SickRage team selects 10 names and everybody votes on them.

**Comment by ratoaq2** [comment]
I like small names (easy to type, easy to say, easy to remember):

How about `seenit`

_**Your friend:** Have you seen the season finale of ...?_
_**You:** Of course, I have `SeenIt`_  :open_mouth:

**Comment by labrys** [comment]
@ratoaq2 too close to [Scene It?](https://en.wikipedia.org/wiki/Scene_It%3F)

**Comment by Antioch18** [comment]
I was thinking something descriptive about SR that excludes anything TV specific. SR is essentially an organizer/tracker/agent that does things for you. So my mind wandered a bit through this area...

- Butler
- Footman
- Attendant
- Steward
- Assistant
- Maid
- Agent

Pair it with a descriptor: Media, Bit, Video, and you get something like:

- BitButler
- MediaMaid
- VideoAgent

I also through about my good, old favorite: MCP - Master Control Program. Or just MC - Master Control (if you get the reference...).

Or you could go with just random funky nouns like other software such as "Hand Break" and "Guacamole."

- Flapjack (Logo: a nice stack of pancakes dripping with butter and syrup -- now I'm hungry)
- Blender
- Jalapeno
- Scrambled Egg (Logo: a "crazy" egg, think the original sickbeard logo but an egg face)
- Thunderstorm
- Typhoon
- Rattlesnake
- Black Mamba, Green Mamba
- Sequoia
- Pinecone
- Sugarcane
- Electric Eel
etc.

Hopefully this brainstorm helped trigger ideas in others.

**Comment by fugglepoop** [comment]
Fredom.   Sounds like freedom.  It's the Middle English spelling. Google will autocorrect to freedom unless you click on 'search for fredom instead'. Those who know what they want can find it.   Urban dictionary reckons in Norwegian it means : something which cannot be defeated.

**Comment by paidhima** [comment]
I like **Remedia** (ReMedia?). "Re-" implies "to do again", "media" is obvious but maybe not such a cliched use. Remedia also invokes "remediation", "remedy", and "remedial". Each of those words implies active improvement. A quick Google search of "remedial" gives me "done to correct or improve something : done to make something better". That describes what SR does rather well, without describing _how_ it does it in such a way as to draw specific attention. You might even find meaning in "medi", the Latin root for "middle". I would certainly consider SR a middleman of sorts, bringing together users and the various involved services while keeping (most of!) the messy bits hidden.

**Comment by bobbysteel** [comment]
How does pymedusa fit into all this now?

**Comment by miigotu** [comment]
@bobbysteel pymedusa is unaffiliated with this project

**Comment by h4wkmoon** [comment]
i like the japanese names: 
Hige 
Terebi
Kibô
Hakike 
Ōtomaton

Or italian : 
Pazzo
